README.md

```diff
@@ -48,12 +48,17 @@ cp firmware/configs/sha2017_defconfig firmware/sdkconfig
 ```
 
 # Build instructions
+Set the path to esp32-toolchain (you have to repeat that on every new terminal):
+```
+export PATH="$PATH:/.../ESP32-platform-firmware/xtensa-esp32-elf/bin"
```

Comment:
What does the ... do? If it's a "fake" path then please replace it with something making that obvious :-)

For example:
export PATH="$PATH:/path/to/my/toolchain/xtensa-esp32-elf/bin"